riscv-sbi.adoc

```diff
@@ -1674,6 +1675,250 @@ The possible error codes returned in `sbiret.error` are shown in the
+| EVIDENCE_FORMATS      | 5              | Bitmap         | 1                     a| Supported evidence X.509 extension identifier bitmap:
+
+                                                                                     * Bit 0: https://trustedcomputinggroup.org/wp-content/uploads/TCG_DICE_Attestation_Architecture_r22_02dec2020.pdf[`DiceTcbInfo`]
+                                                                                     * Bit 1: https://trustedcomputinggroup.org/wp-content/uploads/TCG_DICE_Attestation_Architecture_r22_02dec2020.pdf[`DiceMultiTcbInfo`]
+                                                                                     * Bit 2: https://pigweed.googlesource.com/open-dice/[`OpenDice`]
+| NUM_SMSMT_REGS        | 6             | Integer        | 1                       | Number of static measurement registers
+
+                                                                                     Static measurement registers can not be extended.
+                                                                                     This number must be between 1 and 16 (inclusive).
+| NUM_RMSMT_REGS        | 7             | Integer        | 1                       | Number of runtime measurement registers
+
+                                                                                     Runtime measurement registers can be extended
+                                                                                     with the <<measurement-extend-function>> function.
+                                                                                     This number must be between 0 and 16 (inclusive).
+| MSMT_REGS_INFO        | 8             | MSMT_REG array | 4 * (NUM_SMSMT_REGS + NUM_RMSMT_REGS)   | Measurement registers description array as described in <<table_measurement_register_descriptor>>.
+|===
+
+The `ATST_CAPS` fields are:
+
+* `TCB_SVN`: The Security Version Number (SVN) linked to the reported TCB.
+* `HASH_ALGORTHMS`: A bitmap of all supported measurement hash algorithms.
+  A <<measurement-extend-function>> call must pass one of the supported
+  algorithms.
+* `EVIDENCE_FORMATS`: A bitmap of all supported attestation evidence X.509
+extensions format. A <<get-evidence-function>> call must pass one of the
+supported formats.
+* `NUM_SMSMT_REGS`: The number of static, non extensible, measurement registers.
+These are typically used by the platform to measure all boot components.
+* `NUM_RMSMT_REGS`: The number of runtime extensible measurement registers.
+These are used to store measurements created and passed by the workload itself,
+through a <<measurement-extend-function>> call. This is set to 0 if the platform
+does not allow for extending the TCB  measurement.
+* `MSMT_REGS_INFO`: An array of measurement register descriptors, as defined by
+<<table_measurement_register_format>>. The size of this array is dynamic as it
```

Comment:
Nit: Replace "as it", with "..., and depends on  on the number of static and runtime registers.".